Modules/Verification.cs

```diff
@@ -0,0 +1,288 @@
+
+	    private readonly ConcurrentDictionary<string, HashedValue> HashedValues = new ConcurrentDictionary<string, HashedValue>();
+
+        public async Task<List<Command<TUser>>> Init<TUser>(IBotwinderClient<TUser> iClient)
+            where TUser : UserData, new()
+        {
+            BotwinderClient<TUser> client = iClient as BotwinderClient<TUser>;
+            List<Command<TUser>> commands = new List<Command<TUser>>();
+
+//!verify
+            Command<TUser> newCommand = new Command<TUser>("verify");
+            newCommand.Type = CommandType.Standard;
+            newCommand.RequiredPermissions = PermissionType.Everyone;
+            newCommand.SendTyping = true;
+            newCommand.Description =
+                "This will send you some info about verification. You can use this with a parameter to send the info to your friend - you have to @mention them.";
+            newCommand.OnExecute += async e =>
+            {
+                if ((!e.Server.Config.VerificationEnabled || e.Server.Config.VerifyRoleId == 0) &&
+                    !client.IsGlobalAdmin(e.Message.Author.Id))
+                {
+                    await e.Message.Channel.SendMessageSafe("Verification is disabled on this server.");
+                    return;
+                }
+
+                SocketUser user = null;
+                Server<UserData> server = e.Server as Server<UserData>;
+
+	            // GlobalAdmin verified someone somewhere.
+				if( e.MessageArgs != null && e.MessageArgs.Length == 3 && client.IsGlobalAdmin(e.Message.Author.Id) )
+				{
+					if( e.Message.Content == this.LastVerifyMessage )
+						return;
+
+					this.LastVerifyMessage = e.Message.Content;
+
+					await VerifyUser(user, client.GetServerData(serverID), (e.MessageArgs[2] == "force" ? null : e.MessageArgs[2]));
+					await e.Message.Channel.SendMessageSafe(string.Format(VerifyDone, user.Id));
+					return;
+				}
+
+				// Admin verified someone.
+				if( e.MessageArgs != null && e.MessageArgs.Length == 2 && server.IsAdmin(e.Message.Author as SocketGuildUser) )
+				{
+					if( e.Message.Content == this.LastVerifyMessage )
+						return;
+
+					this.LastVerifyMessage = e.Message.Content;
+
+					guid id;
+					if( (e.Message.MentionedUsers.Count() != 1 || (user = e.Message.MentionedUsers.ElementAt(0)) == null) &&
+					    (!guid.TryParse(e.MessageArgs[0], out id) || (user = e.Server.Guild.GetUser(id)) == null) )
+					{
+						await e.Message.Channel.SendMessageSafe(UserNotFound);
+						return;
+					}
+
+					await VerifyUser(user, server, (e.MessageArgs[1] == "force" ? null : e.MessageArgs[1]));
+					await e.Message.Channel.SendMessageSafe(string.Format(VerifyDone, user.Id));
+					return;
+				}
+
+				// Verify the author.
+				if( string.IsNullOrEmpty(e.TrimmedMessage) )
+				{
+					user = e.Message.Author;
+				}
+
+				// Verify mentioned user.
+				if( e.Message.MentionedUsers != null && e.Message.MentionedUsers.Count() == 1 )
```

Comment:
You can use client.getmentioneduserdata or whatever is it called... and call the method for every one of them.

Reply:
If getmentionedthingmethod requires dbContext param (dont remember) then create the database here, and pass it as parameter, then pass it as parameter into all the subsequent methods that need to touch the database, and use this one.